pkg/cmd/pulumi/deployment_settings_utils.go

```diff
@@ -231,3 +238,11 @@ func ValidateShortInput(s string) error {
 
 	return nil
 }
+
+func convertToLinuxPath(dir string) string {
```

Comment:
This is a bit pedantic, but I think this should be called `convertToUnixPath`.

Reply:
sure